### ISSUE TITLE
spaceBetweenParticles

### ISSUE THREAD

**Issue by abdibaker** [vadimkorr/svelte-carousel#125]
Will be nice if there is a props like spaceBetweenParticles to give space between particles when you get dynamic data

**Comment by darrenpaul** [comment]
Is there any movement on this feature, would be a great addition

**Comment by thedivtagguy** [comment]
Yeah seconded! That's the only thing breaking things right now, otherwise it's the perfect component.

**Comment by MikeeNachtigall** [comment]
There was a similar or even the same issue #145. I added this feature in pull request number #164, so just try to suppress the original author to merge it and you can start using this feature. Thanks!
